app/assets/javascripts/micro_cms.js

```diff
@@ -44,9 +44,11 @@
     }
   };
 
-  window.addEventListener('load', setupMicroCms);
-
   if ('Turbolinks' in window && Turbolinks.supported) {
     $(document).on('turbolinks:load', setupMicroCms);
```

Comment:
I assume that before this change, `setupMicroCms` was executed twice on each page load